### PR TITLE
add lychee package

### LINKED ISSUE #10568: [Wolfi Package Request]: lychee
### What software would you like us to add to wolfi-os. Ideally include a URL to the project and its source.

https://github.com/lycheeverse/lychee 

lychee is a fast link checker. 

### which versions of the software should we include?

latest or 0.13.0

### Add some justification for why this specific package and versions are important.

This will be useful in cases where one wants to check broken links in a repo. 

### Please check all that apply

- [X] This package has an un-restrictive license
- [X] The package/versions proposed are actively maintained upstream
- [X] I am interested in adding this package to Wolfi OS myself
- [X] I am willing to help maintain this package

### PR DESCRIPTION
**Pull request by kranurag7**
Fixes: #10568 

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

**Review by rawlingsj — DISMISSED** [review]

**Comment by kranurag7** [comment]
@rawlingsj I force pushed after fixing the lint errors using https://github.com/chainguard-dev/yam

File: lychee.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: lychee
+  version: 0.13.0
+  epoch: 0
+  description: "Fast, async, stream-based link checker written in Rust"
+  copyright:
+    - license: MIT
```
Comment (rawlingsj) [inline lychee.yaml:7]:
```suggestion
    - license: Apache-2.0 AND MIT
```

**Comment by rawlingsj** [comment]
Thanks @kranurag7 - I did just double check the licence and noticed lychee contains both MIT and Apache 2.  I added a fix suggestion above.

**Comment by rawlingsj** [comment]
Just a thought, I wonder if we'd have a name clash if / when we package https://github.com/LycheeOrg/Lychee.  I don't think it's a blocker, maybe we package https://github.com/LycheeOrg/Lychee as lychee-pms-5 (photo-management-system) or something?  

It's not a blocker as such but just wanted to as the question in case we wanted to name this package differently?

**Comment by rawlingsj** [comment]
SGTM! Thanks again for the contribution @kranurag7

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**